content/influxdb/v2.0/upgrade/v1-to-v2.md

```diff
@@ -12,33 +12,155 @@ aliases:
 
-{{% warn %}}
-Back up all data before upgrading with `influx upgrade`.
-{{% /warn %}}
+<!-- TODO summarize effects on other resources -->
+
+## Before you begin: important considerations
+
+Before upgrading to InfluxDB 2.0, please consider the following guidelines;
+some or all might apply to your specific installation and use case.
+The sections below contain our recommendations for addressing possible gaps in the upgrade process.
+You may find while reading that you'd like to hold off on upgrading for now.
+
+### Continuous queries
+
+When upgrading from InfluxDB 1.x and using continuous queries (CQs) to downsample data,
+we recommend extracting CQs from your InfluxDB 1.x instance with `show continuous queries` prior to upgrade.
+Run the following:
+
+{{< keep-url >}}
+```sh
+$ influx
+Connected to http://localhost:8086 version 1.8.3
+InfluxDB shell version: 1.8.3
+> show continuous queries
+```
+
+Save this output so it can be used later.
+
+Continuous Queries are replaced by Tasks in InfluxDB 2.0
+To convert your CQs to Flux, see the [Tasks documentation](/influxdb/v2.0/process-data/get-started/).
+
+### Supported protocols
+
+If you're using one of the [supported protocols](/influxdb/v1.8/supported_protocols/) in InfluxDB 1.x (CollectD, Graphite, OpenTSDB, Prometheus, UDP),
+you may want to research using Telegraf instead.
+InfluxDB 2.0 doesn't support these protocols directly.
+Use Telegraf to translate from the source protocol to InfluxDB 2.0.
+
+### Kapacitor
+
+<!-- TODO edit this section -->
+
+There is no [subscriptions API](/influxdb/v1.8/administration/subscription-management/) in InfluxDB 2.0.
+To continue using Kapacitor, update the Kapacitor configuration to use the appropriate security credentials per [1.x compatibility API docs 1](/influxdb/v2.0/reference/api/influxdb-1x/)
+(using the appropriate user and token to allow access).
+
+<!-- from TH:-->
+
+OSS users can continue to use Kapacitor with InfluxDB OSS 2.0 under the following scenarios:
+
+1. Kapacitor Batch-style TICKscripts work with the 1.x read compatible API
+   Existing Kapacitor user credentials should continue to work using the [1.x compatibility API](/influxdb/v2.0/reference/api/influxdb-1x/).
+2. Kapacitor Stream-style TICKscripts will not work using the subscription API (as there isn’t one present in 2.0).
+   We recommend writing data directly to *both* Kapacitor and InfluxDB.
+   This allows stream tasks to continue to work.
+   (See dual writes using Telegraf.)
+   If you use other mechanisms to feed data in, Telegraf could be used as an interstiatial layer to feed both Kapacitor and InfluxDB.
+   There are other architectures and mechanisms possible.
+
+### User migrations
+
+`influxd upgrade` attempts to migrate existing 1.x users and their permissions.
+However, *administrative users are not migrated*.
+
+If using an admin user for vizualition or Chronograf administrative functions, you may want to create a new read only user before upgrading.
```

Comment:
If you

Reply:
I would also add that they should login via the CLI to influxdb and perform a `show users` meta query....any user labeled "admin" will NOT be upgraded.